receiver/mysqlreceiver/client.go

```diff
@@ -0,0 +1,89 @@
+// Copyright  OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mysqlreceiver
+
+import (
+	"database/sql"
+	"fmt"
+
+	// registers the mysql driver
+	_ "github.com/go-sql-driver/mysql"
+)
+
+type client interface {
+	getGlobalStats() (map[string]string, error)
+	getInnodbStats() (map[string]string, error)
+	Close() error
+}
+
+type mySQLClient struct {
```

Comment:
Nit: some names in this file contain `mysql`, which also appears in the package name. Consider renaming them to avoid repetition (e.g., `mySQLConfig -> config`, `mySQLClient -> client`).

Reply:
Both `config` and `client` are already used or imported into the package, so these names were chosen to differentiate from those.